src/test/data/XmlUtilTest/validRoom.xml

```diff
@@ -10,8 +10,8 @@
             <tagged>vip</tagged>
         </guest>
         <bookingPeriod>
-            <startDate>29/10/2018</startDate>
-            <endDate>4/11/2018</endDate>
+            <startDate>30/10/2018</startDate>
```

Comment:
Same as earlier, should not have to push this file every PR o.o

Reply:
the saveDataToFileIfMissing is supposed to fix this. see if it works tomorrow